pkg/web/static/js/constants.js

```diff
@@ -303,25 +303,56 @@ export const IDENTIFIER_TYPE = {
   NATURAL_PERSON_NAME_TYPE_CODE_MISC: 'Unspecified',
 };
 
-// Create arrays for each identifier type category from the IDENTIFIER_TYPE object.
-export const addressTypeArray = Object.entries(IDENTIFIER_TYPE)
-  .filter(([key]) => key.includes('ADDRESS_TYPE_CODE'))
+const ADDRESS_TYPE = {
+  HOME: 'Residential',
```

Comment:
The object above was previously used to set the values and options in the dropdowns for each preview form. It's currently used elsewhere, so it will be removed in a follow on story.